ru/mail/chrome/communicator/utilityOverlay.dtd

```diff
@@ -52,7 +52,7 @@
 <!ENTITY pac.label                                      "Онлайн (прокси: авто)">
 <!ENTITY pac.accesskey                                  "a">
 
-<!ENTITY proxy.label                                    "Конфигурация прокси">
+<!ENTITY proxy.label                                    "Конфигурация прокси…">
```

Comment:
Yes, it is. What about a shorter "Настройка прокси-сервера" version or simply "Прокси-сервер". Full form is much more common in UIs, though every user is likely to easily understand the short one. It just looks like we mix PC user slang terms with strictly official translations taken from Microsoft Language Portal, which is a great approach as such. It's like saying "мамка" instead of "материнская плата". Or "проц".